lib/modules/userTagger.js

```diff
@@ -337,7 +393,7 @@ function populateDialog(tag: Tag, card) {
 				<label class="fieldPair-label" for="userTaggerColor">Color</label>
 				<select id="userTaggerColor" value="${tag.color}">
 					${colors.map(({ textColor, color }) => string._html`
-						<option style="color: ${textColor}; background-color: ${color}" value="${color}">${color}</option>
+						<option style="color: ${textColor}; background-color: ${color}" value="${color}" ${tag.color === color && string._html`selected`}>${color}</option>
 					`)}
 				</select>
 			</div>
```

Comment:
It would be neat to add a presets directly this pop-up. Perhaps a "save preset" button add be located next to the preview?

Reply:
Would "save as preset" also save that tag to the current selected user, or would it only append that to the presets in the dialog and the presets table in the settings console?

Reply:
I think the latter sounds most intuitive.